backend/services/syson-direct-edit-grammar/src/main/resources/DirectEdit.g4

```diff
@@ -119,6 +119,12 @@ valueExpression :
 	'=' literalExpression ('[' measurementExpression ']')?
 ;
 
+
+featureValue :
```

Comment:
I would prefer you change `valueExpression` here instead of creating a new rule `featureValue`
And add a new "somethingExpression" for the existing `'=' literalExpression ('[' measurementExpression ']')?` which seems to only be used by `guardExpression` after your changes